app/assets/stylesheets/pages/DetailedWork.scss

```
@@ -0,0 +1,13 @@
+.artwork-description::-webkit-scrollbar {
+    width: 12px;
+    height: 12px;
+    }
+    
+.artwork-description::-webkit-scrollbar-thumb {
+background: $magenta;  
+border-radius: 10px;
+}
+
+.artwork-description::-webkit-scrollbar-thumb:hover {
+background: $magenta;  
```

You can try using a SASS function to get a different color on hover!

```suggestion
    background: darken($magenta, 10%);  
```